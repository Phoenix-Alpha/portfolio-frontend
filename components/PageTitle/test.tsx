import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { PageTitle } from '.'

describe('<PageTitle />', () => {
  it('should renders correct <PageTitle />', () => {
    const wrapper = shallow(<PageTitle title="Page Title" />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
