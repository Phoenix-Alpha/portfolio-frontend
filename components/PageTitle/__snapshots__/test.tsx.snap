// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageTitle /> should renders correct <PageTitle /> 1`] = `
<h1
  className="mb-4 font-display text-2xl lg:text-5xl font-black text-white"
>
  Page Title
</h1>
`;
